### ISSUE TITLE
latest @assistant-ui/react is incompatible with latest @assistant-ui/react-ai-sdk

### ISSUE THREAD

**Issue by Yhozen** [Yonom/assistant-ui#974]
Basically latest versions are incompatible because the provider is expecting the runtime to implement more methods
```package.json
"@assistant-ui/react": "0.5.75",
"@assistant-ui/react-ai-sdk": "0.6.11",
```

```
Type error: Type 'AssistantRuntimeImpl<ThreadRuntimeImpl>' is not assignable to type 'AssistantRuntime'.
  Types of property 'thread' are incompatible.
    Type 'ThreadRuntimeImpl' is missing the following properties from type 'ThreadRuntime': stopSpeaking, speech
  76 |
  77 |   return (
> 78 |     <AssistantRuntimeProvider runtime={runtime}>
```

the current workaround is using a previous version of the react package. ("@assistant-ui/react": "0.5.74")

**Comment by Yonom** [comment]
Thanks for reporting this! I have found the source of this issue and will fix it

**Comment by Yonom** [comment]
@Yhozen this issue has been fixed in the newest versions (v0.5.76 and v0.6.12)
